website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,13 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+
+        var cookies = document.cookie.split(';');
+        for (var i = 0; i < cookies.length; i++) {
+          var cookieName = cookies[i].match(/([^=]+)(?=\=)/)[0];
+          document.cookie = cookieName + '= ; expires=Thu Jan 1 00:00:00 1970 GMT; path=/' + 
```

Comment:
Oops, the trailing `+` should not be there, [but](http://www.quirksmode.org/js/cookies.html) [every](http://www.java2s.com/Tutorial/JavaScript/0080__Development/Deleteallcookies.htm) [solution](http://stackoverflow.com/questions/179355/clearing-all-cookies-with-javascript) [I](http://stackoverflow.com/questions/595228/how-can-i-delete-all-cookies-with-javascript) [have](http://stackoverflow.com/questions/28804564/delete-all-cookies-of-browser-using-javascript) [found](https://www.reddit.com/r/javascript/comments/2vafdu/how_to_remove_cookies_with_javascript/) either only clears a cookie (meaning it would need to be looped) or has a loop in it already.